src/App.tsx

```diff
@@ -1,7 +1,35 @@
-import React from 'react';
+import { Component } from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  message: string,
+};
+
+export class App extends Component<{}, State> {
+  state: Readonly<State> = {
+    message: 'Nothing was pressed yet',
+  };
+
+  componentDidMount() {
+    document.addEventListener('keyup', this.pressedKey);
+  }
+
+  componentWillUnmount() {
+    document.removeEventListener('keydown', this.pressedKey);
+  }
+
+  pressedKey = (event: KeyboardEvent) => {
```

Comment:
The name of functions should start with **verb.**
```suggestion
  pressKey = (event: KeyboardEvent) => {
```